### DISCUSSION TITLE
Add Advanced Filter to Relational Drawer Listings

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#5743] [Archive]
When I have a relationship and I want on related to show only specific ones, like do not show the ones with status `archived`, how can I do that?

<img width="991" alt="Screen Shot 2021-05-18 at 10 11 41" src="https://user-images.githubusercontent.com/4686040/118607611-74bc4a00-b7c1-11eb-8c67-d95c6c93c5c7.png">

Thx.

**Comment by benhaynes** [comment]
Since this is in a drawer, I assume you're looking for a way to do this on a relational interface (eg: Select Item)? It's possible on the main collection page, but I don't think we have advanced filtering on the drawers yet.

**Reply by botzill** [reply]
Yes, having this on a relational interface would be easier. I guess there can be added a JSON editor where we are able to set more advanced filters:

```json
"status": {
  "_neq": "archived"
}
```
?

**Reply by rijkvanzanten** [reply]
I believe there is an open feature request/issue for that already 🤔

**Reply by rijkvanzanten** [reply]
Ah yeah here it is: https://github.com/directus/directus/discussions/5185

**Reply by benhaynes** [reply]
Maybe I misunderstood what @botzill was requesting, but I see these as different. The option in #5185 is for a global filter set by the admin... but I was thinking this one was about users having the ability to filter in the drawer so they can find a specific item to select.

**Reply by benhaynes** [reply]
Both are useful!

**Reply by rijkvanzanten** [reply]
Yeah! The "I want on related to show only specific ones" made me think it was just the "ahead of time, I would like to configure the interface to only ever show XXX" part 🙂
